components/suggestion/command_provider/app_command_parser/app_command_parser.ts

```diff
@@ -1136,12 +1226,7 @@ export class AppCommandParser {
         case ParseState.TickValue:
             return this.getValueSuggestions(parsed, '`');
         case ParseState.Rest: {
-            const execute = getExecuteSuggestion(parsed);
```

Comment:
Can you detail why was this removed?

Reply:
I did this wrong when I was doing the rest argument. It was working, but conceptually was wrong. The execute suggestion is added somewhere else, and depending on the state. I made this change so the execute suggestion is always added in the same place, and there is no chance of repetition.